app/landing.jade

```diff
@@ -148,7 +148,7 @@
     form.form-horizontal(
       role="form"
       name="signupForm"
-      ng-submit="signup()"
+      ng-submit="signup(fields.email)"
```

Comment:
This actually achieves nothing, because in the controller `$scope.signup = () =>` doesn't have any parameters. It gets `fields.email` directly from `$scope`.